release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 13 November 2025 - Fix - Admin - Usage
+
+Fixed the issue where selecting the deleted systems filter in the organization usage page displayed no usage data.
```

Comment:
Could you rephrase the title so it's a short description of the issue, and also rephrase the description so it is more in line with that of other fixes? Right now, it's also not entirely clear what you mean with "the deleted systems filter", so I'm not sure how best to rephrase this myself.